src/vivarium_ciff_sam/model_specifications/ciff_sam.yaml

```diff
@@ -58,8 +60,8 @@ configuration:
     wasting_equations:
         include_mortality: False
 
-#    screening_algorithm:
-#        scenario: 'baseline'
+    treatment:
```

Comment:
it might be clearer in the long run to call this "intervention_implementation", since the "intervention technologies" are considered to be both "treatment" and "prevention".

Reply:
changed to intervention